lib/chef-dk/command/generator_commands.rb

```diff
@@ -37,19 +37,22 @@ module SharedGeneratorOptions
         :short => "-I LICENSE",
         :long => "--license LICENSE",
         :description => "all_rights, apache2, mit, gplv2, gplv3 - defaults to all_rights",
-        :default => "all_rights"
+        :default => "all_rights",
```

Comment:
I think it would be better to do `:default => Chef::Config[:knife][:cookbook_license] || "all_rights"` instead of the proc - thoughts?